examples/taco/nodejs/.env.example

```diff
@@ -1,2 +1,2 @@
-RPC_PROVIDER_URL=
-PRIVATE_KEY=
+RPC_PROVIDER_URL=https://polygon-mumbai.infura.io/v3/3747007a284045d483c342fb39889a30
+PRIVATE_KEY=0x900edb9e8214b2353f82aa195e915128f419a92cfb8bbc0f4784f10ef4112b86
```

Comment:
whoops

Reply:
They were already "leaked" in the `lynx.yml` CI job.

Reply:
That's true, but repeating it will make it more difficult to cleanup later. Since this is mean to be an example, imo it's suitable to have `.env.example` or `env.template` without values filled in.

Reply:
Ah, I see you did not repeat it, it's relocated.  Disregard

Reply:
One of the objectives of https://github.com/nucypher/nucypher-ts/issues/317 is to provide premade Enricos. Any ideas how to achieve that without putting stuff into repo? Adding premade Enrico's to

Reply:
Ok, solved